SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -153,6 +153,16 @@ void ActorSystem::ProcessUpdates(const FEntitySubViewUpdate& SubViewUpdate)
 				ApplyComponentAdd(Delta.EntityId, Change.ComponentId, Change.Data);
 				ComponentAdded(Delta.EntityId, Change.ComponentId, Change.Data, ToResolveOps);
 			}
+			for (const ComponentChange& Change : Delta.ComponentsRemoved)
+			{
+				// NOTE: We have to deal the with dormant before ComponentUpdated
```

Comment:
So in the end you've decided to not handle this at a higher level? I'd imagined finding `DORMANT_COMPONENT_IDs` in `ComponentsRemoved` before going into this loop or something.

Reply:
We could split them in to two before that is true. Let's see if people are happy with the overall approach and if so I can add that.